src/odemis/acq/stream/_static.py

```diff
@@ -337,7 +337,7 @@ def __init__(self, name, raw, *args, **kwargs):
 
         # colouration of the image
         tint = raw.metadata.get(model.MD_USER_TINT, default_tint)
-        self.tint.value = tint
+        self.tint.value = img.md_format_to_tint(tint)
```

Comment:
Please catch `ValueError` here, and in such case, use default_tint as tint. This ensures that if it's a colormap name, and for some reason matplotlib doesn't know this name, it's still possible to open the file. maybe you could write it this way:
```python
tint = raw.metadata.get(model.MD_USER_TINT, default_tint)
try:
     tint  = img.md_format_to_tint(tint)
except ValueError as ex:
   logging.warning("Failed to find tint: %s", ex)
```